newEvaluation.html

```diff
@@ -0,0 +1,394 @@
+				</td>
+			</tr>
+		</table>
+	</section>
+	
+	<!-- Section3 -->
+	<section>
+		<h5>Should the candidate be hired?</h5>
+		<textarea rows = "6" cols = "" placeholder = "The type of project that is suitable for the candidate.&#10;Is guidance required for the candidate"></textarea>
+	</section>
+	
+	<!-- Section4 -->
+	<section>
+		<h5>General Impression</h5>
+		<textarea rows = "6" cols = "" placeholder = "*required" required></textarea>
+	</section>
+	
+	<!-- Section5 -->
+	<section>
+		<h5>Workflow, Leadership & Soft Skills</h5>
+		<textarea rows = "5" cols = "" placeholder = "Describe the environment in which the candidate works.&#10;Describe any guidance or management experience."></textarea>
+	</section>
+	
+	<!-- Section6 -->
+	<section>
+			<fieldset>
+				<legend>OOP, Design Patterns</legend>
+				<ul>
+					<li>
+						<label>Classes</label>
+						<select name = "classes">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Exception handling</label>
+						<select name = "exceptionHandling">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Version Control</label>
+						<select name = "versionControl">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Access modifiers</label>
+						<select name = "accessModifiers">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Design Patterns</label>
+						<select name = "designPatterns">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Issue Tracking</label>
+						<select name = "issueTracking">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Polymorphism</label>
+						<select name = "polymorphism">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>RegEx</label>
+						<select name = "regEx">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+				</ul>
+			</fieldset>
+			
+			<fieldset>
+				<legend>HTTP</legend>
+				<ul>
+					<li>
+						<label>Protocol</label>
+						<select name = "protocol">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Response codes</label>
+						<select name = "responseCodes">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>REST</label>
+						<select name = "rest">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Headers</label>
+						<select name = "headers">
+							<option disabled>Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Request methods</label>
+						<select name = "requestMethods">
+							<option disabled">Evaluation</option>
+						</select>
+					</li>
+					<li>
+						<label>Sessions & Cookies</label>
```

Comment:
Let's do a full audit of all such unescaped HTML entities and replace throughout. Kthxbai! :smile: